### PR TITLE
Add: Added Wishlist Page

### PR DESCRIPTION
**Pull request by JaswanthReddyPadala**
Added WishList Page

image

### CODE REVIEW

File: qmart.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
 @import url(./styles/products.css);
 @import url(./styles/login.css);
 @import url(./styles/cart.css);
+@import url(./styles/wishlist.css);
 
 *,
```
Comment (ckahub) [inline qmart.css:9]: remove uncessary * & below * also

**Review by ckahub — COMMENTED** [review]
code is clean

**Review by shivampandey0 — APPROVED** [review]
Good looks good and is easy to understand with the use of CSS variables.

**Review by MaruthiPothuganti — COMMENTED** [review]
Love the UI design. Code looks clean and good to merge

**Pull request merged**